### ISSUE TITLE
Ideas for improving accessibility TO and accessibility FROM monographic research

### ISSUE THREAD

**Issue by NateShoobs** [mjy/monographers_do#1]
Matt, this is a really cool idea, thank you for putting it together. I think it touches on an underlying principal that a lot of accessibility stuff can be tied to: namely that the taxonomic workflow itself is largely idiosyncratic, fractured, and nonstandard. 

What I mean is that taxonomists working on monographs are often working on their own, in isolation, in the way that they learned how to work (most of us are self taught and gain opinions about the ‘ideal’ way to do things by bouncing around different collecting institutions as end users of  and then make up the ‘workflow’ or process of monography as we go). There should be a place we can point that says “Hey! You there, with the microscope/calipers/forceps! Here are all of the resources available to you, here is how you should be writing your monograph/storing your data so that it’s interoperable, easily accessible, etc.” It seems like that is what TaxonWorks is actually designed to be, and that tool is one of the reasons I got interested in participating in these workshops in the first place!

Such a ‘start here, taxonomist!’ set of standards could also provide a standard framework for conducting museum visits, a kind of bottom-up-bionomia approach in collaboration with collections staff:

All taxonomists and museum users should be encouraged by collections staff to provide their unique ORCID/wikidata ID to museum collections for use in visitors logs (a kind of digital check-in, which also helps record biographic/historical data by creating a pre-parsed, universal museum visitors’ log)

Other ideas: Annotations attached via ID —maybe custom labels?

Names of taxonomists in agent / annotation fields in local biodviersity databases linked with unique IDs to improve porting out to Bionomia?

Interoperability with edits in WoRMS/MolluscaBase/AntBase/other authoritative online taxonomic databases? 

I’m kind of raving now, but I do like the approach of coming up with like... ‘a roadmap to a standardized taxonomic workflow’ that stresses/bakes in data interoperability, longevity, and accessibility. (I’ll admit I currently do basically none of the things that you outline on that Git page, Matt!) This would also serve to increase the accessibility of taxonomic work to those who are new to it. 

Connecting everyone regardless of background to the resources that already exist and the ways in which they are used, and ensuring that these resources feed worker’s efforts in the form of data writ-large clearly and cleanly back out into the world. Of course like any standard this would only be as useful as the degree of adoption...but this workshop series and potential paper in the new SSB bulletin seems like a good venue place to propagandize and set out the idealized ‘accessible taxonomic workflow’.

**Comment by sarahjjacobs** [comment]
This made me think of some sort of flow diagram, or a visual map of resources available for various monograph components. So folks can see the 'road ahead' or the birds-eye view, as well as details about each step.

Also, the links to these resources are really useful and I also think Nate is right, that many of these resources might be unfamiliar. I wonder if a set of basic protocols and/or step-specific 'here's how you use/do this' would make it easier for a broader set of people to work in a common format.

**Comment by mjy** [comment]
@sarahjjacobs exactly. It's easy to list things you can do, it's hard to describe how. The "howto" is completely missing for nearly all steps in the README list, at least as accessible to Taxonomists in a federate place.  Much of this was learned on the fly.  Also of note, various bits were done, but for all intents and purposes shouldn't be repeated.  

Anecdotally we have just met yesterday with @JCGiron and about doing this while facilitating a monograph built in TaxonWorks by collaborators in South/Central America.  Super early discussion. I find opportunities where documentation is done while the project is undertaken helps all parties, things can get fixed when they come up, improvements can be suggested etc., its also part of a workflow that is producing more than just documentation, but an actual product.  Perhaps we can open this up to others.

**Comment by sarahjjacobs** [comment]
@mjy I was actually just thinking about this - I couldn't agree more. Building and refining the protocols and workflows during the process is very helpful and helps to organize ones thoughts during the process itself. 
For myself, I'm currently starting my own lab and thinking about the best way to bring together multiple datasets from multiple sources (my collaborators, my phd, random side projects, etc) in a coherent way such that my current (and future) collaborators can all easily contribute, access, and pull information. So, I'm guessing kind of similar to the group of South/Central America. Admittedly, I hadn't yet researched many of these resources, so I'm happy to have these as a starting place. If you do consider opening up that 'monograph building process' to others, I'd be interested in participating and/or learning from the process. That's definitely where I'm headed. Also, whatever protocols and workflows I end up generating, I'd be very happy to share with others. These (your working group experiences, my experiences, others in this revitalizing monographs group?) could be the basis of that roadmap/how-to for many of these resources/tools.

**Comment by mjy** [comment]
@sarahjjacobs our [group](https://speciesfilegroup.org) tries to be as open as possible.  We presently have two weekly zoom meetings (See 'Events' on our webpage) that people are welcome to join, lurk, ask questions.

I'd love to have us start similar events for a monographic process themed effort, particularly for those with specific needs or goals. There are lots of places to dream, less to tackle problems on the ground.  That said, our group does plenty of dreaming as well.  With @debpaul learning the ropes after joining us in the fall (well, she's teaching us ropes too) we hope to be able to develop a significant outreach-specific effort in the long run.

**Comment by teleaslamellatus** [comment]
Are you going to start a third meeting group for Monographs or we should bring up questions at one of the already existing meetings (nomenclature or digitization)?

**Comment by mjy** [comment]
@teleaslamellatus for Species File Group nothing has been confirmed as of yet.  This (Revitalizing Monographs) group is still exploring things, they, and the other groups, have been tasked with a paper for April on some topic that was roughly Accessibility. Join us if you want (see my email re zoom).
